scala-package/pom.xml

```diff
@@ -231,6 +231,7 @@
           <skipTests>${skipTests}</skipTests>
           <reportsDirectory>${project.build.directory}/surefire-reports</reportsDirectory>
           <junitxml>.</junitxml>
+          <stdout>F</stdout>
```

Comment:
is this needed?

Reply:
It brings the whole stacktraces for Scalatest. So yes, needed and good for all developers to debug.